src/PowerSystems.jl

```diff
@@ -412,7 +412,6 @@ export create_poly_cost
 #export make_time_series
 export get_bus_numbers
 export set_bus_number!
-export set_number!  # Remove this in v5.0.
```

Comment:
This method still exists for DCBus.

Reply:
Deleting this export shouldn't be on this PR if this PR is to be merged before the next major version, right?

Reply:
#1276 added this line. I later realized that this function for DCBus was already exported and reverted the change. I suppose you could argue that should have exported the function here in case DCBus is ever changed or deleted. I'm not sure it matters. I did add a test to ensure that it is exported for ACBus.

Reply:
I guess if I were being really nitpicky I'd say it still belongs in this file since (until PSY5) it's still part of the interface even independently of DCBus. But it's not a big deal and I'm okay with it either way.